index.ts

```diff
@@ -253,23 +254,45 @@ export class Project {
     this.pkg.version = value;
   }
 
+  async write() {
+    this.writeProject();
+    await this.binLinks();
+  }
+
+  /**
+   * @deprecated please use `await project.write()` instead.
+   */
   writeSync() {
+    console.warn('this method is deprecated, please use write instead');
```

Comment:
We should probably use your deprecated message from above which includes the exact method people should use:

```
project.writeSync() is now deprecated, please use `await project.write()` instead.
```